docs/FAQ/FAQ-Stores.md

```diff
@@ -24,6 +25,10 @@ To create your first store, go to > Stores from the header menu and click "creat
 
 There's no limit on a number of stores you can create in BTCPay.
 
+## why are invoices without payment showing as complete?
+
+When an invoice is created to receive a payment value of 0 (zero amount due) the invoice by definition, is already paid. The invoice will appear as complete as soon as it's created. 
```

Comment:
It would be good to give more context on why this happens (giveaway, etc)

Reply:
True, ok let me know what you think of this..